app/models/observation/scopes.rb

```diff
@@ -429,7 +429,7 @@ module Observation::Scopes # rubocop:disable Metrics/ModuleLength
           ->(bool = true) { where(specimen: bool) }
 
     scope :has_sequences, lambda { |bool = true|
-      return all unless bool
+      return all if bool.nil?
 
```

Comment:
@mo-nathan Check the other `has_` scopes that call `joined_relation_condition` - none of them have the `nil` guard clause, and I think that's because AR returns `all` if neither condition is met. 

If that's not the case, we need to fix it, although I believe it is the case and is tested for other scopes. I suggest adding a Query test that checks the results of each (`nil`, `true`, `false`) for `has_sequences` to make sure this is the case, and if so, deleting the `nil` check and adding a comment to `joined_relation_condition` making it explicit that it returns `all` on `nil`. (There's nothing wrong with the nil guard, it's just that it's misleading.)